BugDoom - web app/src/main/java/com/infoshareacademy/pl/model/Track.java

```diff
@@ -1,23 +1,33 @@
 package com.infoshareacademy.pl.model;
 
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Positive;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Objects;
-
+@Component
```

Comment:
to nie powinien być Component

Reply:
faktycznie można to pominąć. Pozostało z przenoszenia projektu z aplikacji konsolowej